client/src/partials/budget/analysis/analysis_budget.js

```diff
@@ -360,7 +360,9 @@ angular.module('bhima.controllers')
       });
 
       // Exportation
-      exportFile.csv(fileData, 'Budget Analysis Report');
+      var labelPeriod     = session.selectedPeriod || '';
+      var labelFiscalYear = session.selectedFiscalYear.start_year || '';
```

Comment:
It's less probable to have session.selectedPeriod or session.selectedFiscalYear to undefined but if this case occur, it'll be better to have ' ' for both, and the name of the file will be like "Budget 2015-09-30-14-57-00"